hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -502,6 +551,27 @@ public void close() throws IOException {
     ecChunkBufferCache.release();
   }
 
+  private void handleStripeFailure(int lastStripeSize, int parityCellSize,
```

Comment:
This is not a big problem, but is this method easier to follow written like this:

```
  private void handleStripeFailure(int lastStripeSize, int parityCellSize,
      boolean allocateBlockIfFull)
      throws IOException {
    for (int i = 0; i < RETRIES_ON_STRIPE_WRITE_FAILURE; i++) {
      stripeWriteStatus =
          rewriteStripeToNewBlockGroup(parityCellSize, lastStripeSize,
              allocateBlockIfFull);
      if (stripeWriteStatus == StripeWriteStatus.SUCCESS) {
	  return;
      }
    }
    throw new IOException(
        "Completed max allowed retries " + RETRIES_ON_STRIPE_WRITE_FAILURE
            + " on stripe failures.");
    }
  }
  ```

  It saves allocating a couple of variables and removes one of the two conditions in the while and the final IF statement. It look me a little bit of time to understand this method, so wanted to see how it would look if written like above.

Reply:
I would use config.getEcWriteMaxRetries() if we rename the property and the related methods to this name, and remove the instance variable used here.